policy/src/main/java/brooklyn/enricher/TimeWeightedDeltaEnricher.java

```diff
@@ -41,6 +42,8 @@
  * presented as a units/timeUnit based on the event timing.
  * <p>
  * NB for time (e.g. "total milliseconds consumed") use {@link TimeFractionDeltaEnricher}
+ * <p>
+ * See also {@link YamlTimeWeightedDeltaEnricher} designed for use from YAML.
```

Comment:
If both this and the YAML alternative are going to be used long-term, why not re-use the code? Alternatively deprecate this one?

Analogous comment for `RollingTimeWindowMeanEnricher` would be nice.